lib/telephone_appointments/summary_document_activity.rb

```diff
@@ -2,16 +2,18 @@ module TelephoneAppointments
   class SummaryDocumentActivity
     PATH = '/api/v1/summary_documents'.freeze
 
-    def initialize(appointment_id:, owner_uid:)
+    def initialize(appointment_id:, owner_uid:, message:)
```

Comment:
Technically speaking - this is a breaking change and requires versioning, unless we default this value. I guess since we're the only consumer this isn't an issue but perhaps something to be mindful of in future.